python/geometryApproximation.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'XML PolyData Reader'
+streamlinesvtp = XMLPolyDataReader(FileName=['/home/emma/ttk/ttk-source/ttk-data-emmwel/GroundWater.cdb/streamlines.vtp'])
+
+# create a new 'XML PolyData Reader'
+stonevtp = XMLPolyDataReader(FileName=['/home/emma/ttk/ttk-source/ttk-data-emmwel/GroundWater.cdb/stone.vtp'])
+
+# create a new 'Elevation'
+fakeShadow = Elevation(Input=stonevtp)
+fakeShadow.LowPoint = [0.13294358695786565, 0.08882809227819845, -0.018024881743751362]
+fakeShadow.HighPoint = [0.06742407365289238, 0.018398674549435334, 0.138207120609427]
+
+# create a new 'TTK IcosphereFromObject'
+tTKIcosphereFromObject1 = TTKIcosphereFromObject(Object=stonevtp)
+
+# create a new 'TTK CinemaImaging'
+tTKCinemaImaging1 = TTKCinemaImaging(Dataset=fakeShadow,
+    SamplingGrid=tTKIcosphereFromObject1)
+
+# create a new 'TTK DepthImageBasedGeometryApproximation'
+tTKDepthImageBasedGeometryApproximation1 = TTKDepthImageBasedGeometryApproximation(Input=tTKCinemaImaging1)
+tTKDepthImageBasedGeometryApproximation1.DepthArray = ['POINTS', 'Depth']
+
+# create a new 'Threshold'
+threshold1 = Threshold(Input=tTKDepthImageBasedGeometryApproximation1)
+threshold1.Scalars = ['CELLS', 'TriangleDistortion']
+threshold1.ThresholdRange = [0.0, 0.02]
+
+# create a new 'TTK IcospheresFromPoints'
+tTKIcospheresFromPoints1 = TTKIcospheresFromPoints(Input=tTKIcosphereFromObject1)
+tTKIcospheresFromPoints1.Subdivisions = 1
+tTKIcospheresFromPoints1.Radius = 0.01
+tTKIcospheresFromPoints1.ComputeNormals = 0
+
+SaveData('CameraNodes.vtp', tTKIcospheresFromPoints1)

```

Comment:
I don't believe this output needs to be stored for this example.
(that would simplify the script)